test/TestInterface.jl.save

```diff
@@ -0,0 +1,65 @@
+module TestInterface
```

Comment:
What's this file about?

Reply:
To move these tests to ApproximateGPs or something like this. But I should probably just save it as a gist somewhere

Reply:
Ohhh okay. I can't say that I'm hugely fussed either way, provided it gets moved quickly.